xbmc/addons/kodi-addon-dev-kit/include/kodi/xbmc_pvr_types.h

```diff
@@ -566,6 +566,8 @@ extern "C" {
     int    iChannelUid;                                   /*!< @brief (optional) unique identifier of the channel for this recording. PVR_CHANNEL_INVALID_UID denotes that channel uid is not available. */
     PVR_RECORDING_CHANNEL_TYPE channelType;               /*!< @brief (optional) channel type. Set to PVR_RECORDING_CHANNEL_TYPE_UNKNOWN if the type cannot be determined. */
     char   strFirstAired[PVR_ADDON_DATE_STRING_LENGTH];   /*!< @brief (optional) first aired date of this recording. Used only for display purposes. Specify in W3C date format "YYYY-MM-DD". */
+    unsigned int iFlags;                                  /*!< @brief (optional) bit field of independent flags associated with the recording */
```

Comment:
This line was required temporarily to be compatible with https://github.com/xbmc/xbmc/pull/17515 in my dev env.

Reply:
Shouldn't you remove this line?

Reply:
Yes, it will be removed when this PR gets rebased after the prior one goes it.

Reply:
By prior PR I mean: https://github.com/xbmc/xbmc/pull/17515

Reply:
Okay.